src/wazuh_modules/syscollector/sysInfoLinux.cpp

```diff
@@ -11,11 +11,21 @@
 #include "sysInfo.hpp"
 #include <fstream>
 #include <iostream>
+#include <sys/types.h>
+#include <sys/stat.h>
 #include "stringHelper.h"
+#include "cmdHelper.h"
 
 constexpr auto WM_SYS_HW_DIR{"/sys/class/dmi/id/board_serial"};
 constexpr auto WM_SYS_CPU_DIR{"/proc/cpuinfo"};
 constexpr auto WM_SYS_MEM_DIR{"/proc/meminfo"};
+#define DPKG_PATH "/var/lib/dpkg/"
```

Comment:
Could you please change this define to a constexpr auto ? I think it'd be better to maintain code consistency

Reply:
will do